HOK.Elastic.FileSystemCrawler.ConsoleProgram/Program.cs

```diff
@@ -160,16 +164,16 @@ static async Task<int> Main(string[] args)
                 {
                     throw new ArgumentException($"Incorrect number of arguments passed to the application; instead, received {args.Length} arguments.\r\n\r\n " +
                         $"Argument format is {{Path to executable}} {{job folder}} {{notes}}\r\n" +
-                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().CodeBase} \"d:\\elastic crawl job definitions\\europe\"\r\n" +
-                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().CodeBase} .\\myjob\r\n" +
-                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().CodeBase} myjob \"my notes about the job\""
+                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().Location} \"d:\\elastic crawl job definitions\\europe\"\r\n" +
+                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().Location} .\\myjob\r\n" +
+                        $"Example: {System.Reflection.Assembly.GetExecutingAssembly().Location} myjob \"my notes about the job\""
                         );
                 }
             }
             catch (Exception ex)
             {
                 //todo we should ensure this gets written out somewhere in case the logger never got setup and this is running headless
-                if (ilerror) _il.LogErr("program.main", null, null, ex);
+                if (ilfatal) _il.LogErr("program.main", null, null, ex);
```

Comment:
Do we need to get the logger and log method aligned? i.e. `ilerror` and `LogError` or `ilfatal` and `LogFatal`?

Reply:
Hey Dan, I don't know why there are two log lines that look like they would generate identical output in the logs. I'm going to guess that it's a mistake (on my part I'm sure). I think the fatal error is the right one and LogFatal would be good if we have it but maybe that change can be easily made when we refactor to nlog.